ppr-ui/tests/unit/BaseDialog.spec.ts

```diff
@@ -85,11 +82,21 @@ describe('Base Dialog tests', () => {
     expect(wrapper.findComponent(DialogButtons).exists()).toBe(true)
     expect(wrapper.findComponent(DialogContent).isVisible()).toBe(true)
     expect(wrapper.findComponent(DialogButtons).isVisible()).toBe(true)
-    expect(wrapper.findComponent(DialogContent).vm.$props.setBaseText).toBe('default text')
-    expect(wrapper.findComponent(DialogContent).vm.$props.setExtraText).toEqual([])
-    expect(wrapper.findComponent(DialogContent).vm.$props.setHasContactInfo).toBe(false)
-    expect(wrapper.findComponent(DialogButtons).vm.$props.setAcceptText).toBe('default accept')
-    expect(wrapper.findComponent(DialogButtons).vm.$props.setCancelText).toBe('default cancel')
+    expect(wrapper.findComponent(DialogContent).vm.$props.setBaseText).toBe(
+      'default text'
+    )
+    expect(wrapper.findComponent(DialogContent).vm.$props.setExtraText).toEqual(
+      []
+    )
+    expect(
+      wrapper.findComponent(DialogContent).vm.$props.setHasContactInfo
+    ).toBe(false)
+    expect(wrapper.findComponent(DialogButtons).vm.$props.setAcceptText).toBe(
+      'default accept'
+    )
+    expect(wrapper.findComponent(DialogButtons).vm.$props.setCancelText).toBe(
+      'default cancel'
+    )
```

Comment:
Are all the scenarios like this and below due to the prettier extension or is this reformatting intentional? 

Either way, could you turn off the auto formatter you are using and revert these changes.

Reply:
We should probably adjust prettier.config.js
`printWidth: 80`

Reply:
I'm not sure, whatever you have to do to stop auto reformatting code like this 😆 
It's seems like it's doing more against you than for you at the moment.

Reply:
I think over time different formatters were used and it's kinda messed up (some lines 80, others 120, etc).